server/src/thirdparty/githubAPI/README.md

```diff
@@ -0,0 +1,140 @@
+# Slack API 
+
+## How to integrate
+Use the below url to integrate the user with slack. Here **code** should be unique token to find your user. This token will be sent in appAdded event.
+```
+/tp/slack/integrate/:code
```

Comment:
Should be changed to github?

Reply:
yes i will be removing the readme itself as of now